sources/packages/backend/libs/queue/src/utilities/queue.util.ts

```diff
@@ -0,0 +1,13 @@
+import { BullModuleOptions } from "@nestjs/bull";
+import { Queues, QUEUE_PREFIX } from "../constants/queue.constant";
+
+/**
+ * Builds the bull module options to register queues in a dynamic wau.
+ * @returns bull module options with all existing queues.
+ */
+export const getQueueModules = (): BullModuleOptions[] => {
+  return Object.values(Queues).map<BullModuleOptions>((queue) => ({
+    name: queue.name,
+    prefix: QUEUE_PREFIX,
```

Comment:
so, are these prefixes the same? 
![image](https://user-images.githubusercontent.com/77353155/202809147-523fa0e3-3d4b-4d55-bdcd-40a7b8d6c08f.png)